.storybook/stories/stack-view/stack-view.stories.ts

```diff
@@ -89,6 +99,35 @@ const StackViewAllTemplate: Story = args => ({
   props: args,
 });
 
+const StackViewAllTemplateWithPopOver: StoryFn = args => ({
```

Comment:
Was this included here intentionally? I think this should in a separate PR.

Reply:
This was added in this PR by mistake, It is part of the CDK popover refactoring, it will be raised along with those changes. Reverted the same in this PR.